### DISCUSSION TITLE
Messed up my nixos zed editor install by deleting the config files

### DISCUSSION THREAD

**Discussion by rp1231** [zed-industries/zed#18792] [Feature Requests]
I'm getting this error now:
```
zeditor --foreground
[2024-10-07T00:59:26+05:30 ERROR fs] No such file or directory (os error 2) about ["/home/nixos/.config/github-copilot"]
```

How do i solve this ?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
